Sources/SwiftProtobuf/Google_Protobuf_Duration+Extensions.swift

```diff
@@ -130,18 +132,41 @@ extension Google_Protobuf_Duration: _CustomJSONCodable {
     }
 }
 
-
 extension Google_Protobuf_Duration: ExpressibleByFloatLiteral {
     public typealias FloatLiteralType = Double
 
     public init(floatLiteral value: Double) {
+        // TODO: this rounds the nanos part towards zero, not to the nearest
```

Comment:
Ah, now I follow.  Yea, and we likely need to call it out in the docs out the ranges impact the rounding.